packages/cli/README.md

```diff
@@ -15,6 +16,15 @@ Zowe CLI is a command-line interface that lets you interact with the mainframe i
  - [Run system tests](#run-system-tests)
  - [FAQs](#frequently-asked-questions)
 
+## Early Access Features
+
+To try out early access features, install the "next" release of Zowe CLI (`npm install -g @zowe/cli@next`). This version may receive breaking changes and is intended to gather early feedback on what may become a future LTS release.
+
+For documentation about these features, see these files:
+
+- [Using Global Profile Configuration](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md)
+- [Using Daemon Mode with "zowex"](https://github.com/zowe/zowe-cli/blob/next/zowex/design.md)
```

Comment:
The daemon mode link resulted in 404 "page not found".

Reply:
Currently it's in the "json-config-next" branch at https://github.com/zowe/zowe-cli/blob/json-config-next/zowex/design.md. The link should work for the next branch after #930 is merged.

Reply:
Ok. Thanks.